ci/build.ps1

```diff
@@ -36,7 +36,8 @@ $scoop = (New-Object System.Net.WebClient).DownloadString('https://get.scoop.sh'
   Invoke-Expression $scoop
 }
 
-scoop install diffutils perl
+scoop install diffutils
```

Comment:
http://repo.msys2.org/msys/x86_64/diffutils-3.5-1-x86_64.pkg.tar.xz is unavailable.
pacman downloads from the same site. How are the mingw builds passing?

Reply:
maybe https://github.com/lukesampson/scoop/issues/2722 is relevant